tools/lint/lint.py

```diff
@@ -840,6 +857,13 @@ def process_errors(errors):
 all_paths_lints = [check_css_globally_unique]
 file_lints = [check_regexp_line, check_parsed, check_python_ast, check_script_metadata]
 
+# Don't break users of the lint that don't have git installed.
+try:
+    subprocess.check_output(["git", "--version"])
+    path_lints += [check_git_ignore]
+except subprocess.CalledProcessError:
+    pass
```

Comment:
This could print a warning or something? I'm not sure if that would break anything.